### DISCUSSION TITLE
"Edit Custom CSS" gives me the Bindings settings instead

### DISCUSSION THREAD

**Discussion by Cnote43** [Budibase/budibase#8627] [General]
I must be missing something incredibly simple and obvious. When I click the "Edit Custom CSS" button, this is what I see:

![image](https://user-images.githubusercontent.com/16570699/200955763-17ade5ae-bdb7-44cc-8966-9478920540f6.png)

which seems to be the Bindings screen. Looking at [this discussion](https://github.com/Budibase/budibase/discussions/4331) it seems like I should just be seeing a box where I can enter CSS (though that discussion is 9 months old). I've tried this on multiple components and I always get the Bindings screen. I've looked though a bunch of other discussions and the docs but can't figure it out.

What am I missing?

**Comment by melohagan** [comment]
Hey @Cnote43 

Have you tried entering some inline CSS into the box?

E.g.
```css
color: red;
```

The reason it now shows Handlebars and JavaScript is because the Custom CSS now supports bindings! (I'll update the docs, I don't think they reflect this change)

So if you had some [App state](https://docs.budibase.com/docs/app-state) that stored a css variable then you could do something like:

```css
color: {{ State.Color }}
```

**Reply by Cnote43** [reply]
Thanks, @melohagan! That works for some components. I had been trying the <style> tag based on a response to an older issue.

Related question - is there any way to apply CSS to form components? I'm trying to increase the font-size, but the CSS isn't getting applied.

**Reply by melohagan** [reply]
Unfortunately the Custom CSS is inline, so it will only apply to one div level, which may or may not work.
You may need to build a custom form field: https://docs.budibase.com/docs/custom-component#example-custom-form-field-component

**Reply by Cnote43** [reply]
Got it. Thanks!
